src/applications/bmqstoragetool/m_bmqstoragetool_printer.cpp

```diff
@@ -885,7 +885,7 @@ class JsonLinePrinter : public JsonPrinter {
         d_ostream << "  \"DataFileDetails\": \n";
         m_bmqstoragetool::printDataFileMeta<
             bmqu::JsonPrinter<true, true, 2, 4>,
-            bmqu::JsonPrinter<false, true, 6, 0> >(d_ostream,
+            bmqu::JsonPrinter<false, true, 0, 0> >(d_ostream,
```

Comment:
It seems it is discarded change from PR https://github.com/bloomberg/blazingmq/pull/665.